src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -38,7 +40,7 @@ The most common causes of a poor CLS are:
 **Summary:** Always include `width` and `height` size attributes on your images and video elements. Alternatively, reserve the required space with [CSS aspect ratio boxes](https://css-tricks.com/aspect-ratio-boxes/). This approach ensures that the browser can allocate the correct amount of space in the document while the image is loading.
 
   <figure class="w-figure">
-    <video controls autoplay loop muted class="w-screenshot">
+    <video controls loop muted class="w-screenshot" poster="video-image-dimensions.jpg">
```

Comment:
Opted for a poster + manual playback here to avoid readers downloading 30MB+ of MP4s. Interested in a confirm we're okay with this.

Reply:
that sounds ok to me